### PR TITLE
docs: adding more Helm context

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR adds additional steps for installing Palette through Helm.  You can review the changes below.

CleanShot 2023-04-06 at 10 02 49

## Review Changes

💻 [Preview](https://deploy-preview-1231--docs-spectrocloud.netlify.app/enterprise-version/deploying-palette-with-helm#upgradepalette)

🎫 [Jira Ticket]()

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1231#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (TylerGillson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:337]:
```suggestion
After the installation process, if you need to change any configuration values under `values.config` in the **values.yaml** file, you must use the Palette API. Otherwise, internal system configurations stored in the Kubernetes ConfigMap `configserver-cm` will be updated, but Palette will not apply the new values. Once the initial installation has completed, Palette only applies changes to these configuration values if they submitted via API.
```

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (TylerGillson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:335]:
```suggestion
The values you specified in the file **values.yaml**, more specifically, all the parameter values that fall under the parameter section `values.config` are stored in a ConfigMap titled `configserver-cm`. 
```

**Review by TylerGillson — APPROVED** [review]

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (ritawatson) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:335]:
```suggestion
The values you specified in the **values.yaml** file all fall under the parameter section `values.config` and are stored in the `configserver-cm` ConfigMap. 
```

**Review by ritawatson — APPROVED** [review]
One suggested change. LGTM.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1231#" title="Author self-approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1231#pullrequestreview-1376366310" title="Approved">ritawatson</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1231#pullrequestreview-1376354461" title="Approved">TylerGillson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/12-enterprise-version/03-deploying-palette-with-helm.md
Comment (rahulhazra97) [inline content/docs/12-enterprise-version/03-deploying-palette-with-helm.md:335]: Small Suggestion:
The values you specified in the **values.yaml** file all fall(falls) under the parameter section `values.config` and are(is) stored in the `configserver-cm` ConfigMap.

**Pull request merged**
